python/cudf_polars/cudf_polars/experimental/sort.py

```diff
@@ -435,12 +435,12 @@ def _(
 
     # Check sort keys
     if not all(
-        expr.is_pointwise for expr in traversal([e.value for e in ir.by])
+        isinstance(expr, Col) for expr in traversal([e.value for e in ir.by])
```

Comment:
Not 100% sure about this.  Maybe should just add the column in the Sort rewrite instead?  I thought, groupby also required only names (thus it was copied from there first), but that seems not true.